code/world.dm

```diff
@@ -54,7 +54,7 @@ var/global/list/map_transition_config = MAP_TRANSITION_CONFIG
 
 	data_core = new /datum/datacore()
 
-	spawn(-1)

```

Comment:
spawn -1 is suppose to happen within the same tick **as long** as it's not gone overboard.

It rarely works out that way, but still.